_posts/wasmer-go-embedding-1.0.md

```diff
@@ -0,0 +1,543 @@
+---
+title: 'Wasmer Go embedding 1.0 liftoff'
+excerpt: 'Announcing the immediate availability of the Wasmer Go embedding 1.0 version!'
+date: '2021-02-22T15:35:07.322Z'
+author: Ivan Enderlin
+---
+
+We are delighted to announce the availability of the [Wasmer Go
+embedding 1.0 version](https://github.com/wasmerio/wasmer-go).
+
+A little more than 1.5 years, we first release `wasmer-go`, the Wasmer
```

Comment:
> About 1.5 years ago we released `wasmer-go`, the Wasmer

Reply:
I've included your suggestion, except the `>` for the quote. We aren't quoting anything here :-).